app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,29 @@
+class BookingsController < ApplicationController
+  def new
+    @booking = Booking.new
+    @bathroom = Bathroom.find(params[:bathroom_id])
+    @user = User.find(params[:user_id])
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @bathroom = @booking.bathroom
+    @user = @booking.user
+    if @booking.save
+      redirect_to @bathroom
```

Comment:
Shouldn't it be redirect_to the new booking path?
*redirect_to booking_path(params[:booking_id])